quizroyale/db/db.py

```diff
@@ -0,0 +1,34 @@
+import abc
+from quizroyale.db import models
+from quizroyale.db.errors import NoSuchUserException
+
+
+class DbService(abc.ABC):
+    @abc.abstractmethod
+    def get_users(self) -> list[models.User]:
+        ...
+
+    @abc.abstractmethod
+    def get_user_by_credentials(self,
+                                username: str,
+                                password: str,
+                                ) -> models.User:
+        ...
```

Comment:
```suggestion
    def get_user_by_credentials(
        self,
        username: str,
        password: str,
     ) -> models.User:
        ...
```

Strange formatting, is it Black?

Reply:
It's <79 lines xD. I have no idea how to format it good